tests/python/gaia-ui-tests/gaiatest/visual_components/visual_component.py

```diff
@@ -0,0 +1,7 @@
+from gaiatest.apps.base import PageRegion
+
+
+class VisualComponent(PageRegion):
```

Comment:
So is there a chance for a frame switching to occur within this class?  If so we might run into the problem that we discussed.  (looks like it doesn't happen here though)